### ISSUE TITLE
--cleanup removes the input file when used together with --mode raw

### ISSUE THREAD

**Issue by don1001** [avast-tl/retdec#309]
Hi, decompile.sh help states that
```
--cleanup    Removes temporary files created during the decompilation.
```
I use it with `--mode raw`,  `--select-ranges`  however it also removes input file which is a bit annoying if I want to select different range I need to copy the input file again. Solution is not to use `--clean` option but it is nice to remove the temporary files automatically.

**Comment by s3rvac** [comment]
Hi. Thank you for the report! You are totally right. Steps to reproduce with the current `master`:
```
$ wget https://github.com/avast-tl/retdec-regression-tests/raw/master/features/raw/binaries/raw.raw
$ retdec-decompiler.sh --mode raw --arch arm --endian big --raw-entry-point 0x0 --raw-section-vma 0 --cleanup raw.raw
$ ls raw*
raw.raw.c  raw.raw.c.frontend.dsm
```

**Comment by don1001** [comment]
cool, thx ! BTW. when you're at it you could remove *.frontend.dsm since it's created during the decompilation and not removed.

**Comment by s3rvac** [comment]
In #309, I have proposed a fix (commit b94efd20374ce2b4873ba49bae72a89bc8ca8b27). After my colleague reviews the changes and accepts them, they will be merged and this issue will be fixed.
